content/posts/android-next.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 'Android Developers Rejoice'
+subtitle: 'We have a brand new web wrapper for Android sans Cordova!'
+author: 'dkanada'
+githubusername: 'dkanada'
+date: 2020-08-28T04:16:09-04:00
+justify: 'center'
+---
+
+This will be a net benefit for users and developers alike!
+Everyone will enjoy the native video player spearheaded by [Stampede10343](https://github.com/Stampede10343) and [vitorsemeano](https://github.com/vitorsemeano) over the course of several months.
+[Maxr1998](https://github.com/Maxr1998) valiantly ported the Cordova portions to native Java (and Kotlin) which means Android developers will feel right at home with the new codebase!
+
+<!--more-->
+
+### Migration
+
+The new release will be a drop-in replacement for the deprecated Cordova client, meaning you can just update from GitHub or F-Droid as you would have previously.
```

Comment:
It's a drop-in but you need to enter your server URL again (we can't migrate that from Cordova unfortunately) and because the local storage is gone you need to set your client settings again; authenticate, set theme etc. etc.